sql/trino/TrinoParser.g4

```diff
@@ -170,6 +170,7 @@ statement
     | SHOW_ SESSION_
         (LIKE_ pattern=string_ (ESCAPE_ escape=string_)?)?                 #showSession
     | SET_ SESSION_ qualifiedName EQ_ expression                          #setSession
+    | RESET_ SESSION_ AUTHORIZATION_                                     #resetSessionAuthorization
     | RESET_ SESSION_ qualifiedName                                      #resetSession
```

Comment:
It looks more clear, don't you think?

```suggestion
    | RESET_ SESSION_ (AUTHORIZATION_ | qualifiedName_)                  #resetSession
```

Reply:
The official documentation divides this into two statements. The same for the official grammar file.
![image](https://github.com/antlr/grammars-v4/assets/20572131/098efbec-deb6-41c0-835a-de7da1e41c2e)
![image](https://github.com/antlr/grammars-v4/assets/20572131/916dcdb3-76dc-4cc8-af9c-c5858c1f40c4)